src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -133,6 +134,15 @@ export const AdminVenuePreview: React.FC<AdminVenuePreviewProps> = ({
               text={venue.config?.landingPageConfig.description}
             />
           </div>
+
+          <div>
+            <span className="title">Created At:</span>
+            {venue.createdAt && format(venue.createdAt, "yyyy-MM-dd HH:mm:ss")}
+          </div>
+          <div>
+            <span className="title">Updated At:</span>
+            {venue.updatedAt && format(venue.updatedAt, "yyyy-MM-dd HH:mm:ss")}
+          </div>
```

Comment:
Maybe it's better not to render the text "Created At:" and "Updated At:" too if there is no `venue.createdAt` and `venue.updatedAt`.

Reply:
hmm, i did the same as it's displayed for venues that do not have a description/tagline. The titles are still there in those cases even if the values are empty. So we should change the behavior for both cases or leave it as is. 

what do you think is better? i'd prefer to be more explicit and to show the titles.

![image](https://user-images.githubusercontent.com/63313732/130402474-a4726d30-7ea1-40da-9a07-3b472cc331ac.png)

Reply:
agreed